### PR TITLE
Added support for sending messages to a specific channel.

### PR DESCRIPTION
**Pull request by AndyCJ**
Changed IrcClient, IrcConnection, and thier interfaces to support sending messages to channels.
Changed SignalRMessageBusConnector to always send messages to '#spikelite' for now.

### CODE REVIEW

File: src/SpikeCore/SpikeCore.Irc.IrcDotNet/IrcClient.cs [hunk 1]
```diff
@@ -83,6 +83,6 @@ private void LocalUser_MessageReceived(object sender, IrcMessageEventArgs e)
         private void IrcClient_RawMessageReceived(object sender, IrcRawMessageEventArgs e) => MessageReceived?.Invoke($"RAW: {e.RawContent}");
         
         // TODO: [Kog 9/17/2018] - Need to wire this back through the UI, which is kinda broken right now anyway...
```
Comment (kog) [inline src/SpikeCore/SpikeCore.Irc.IrcDotNet/IrcClient.cs]: Please remove TODO. It's TODONE.

File: src/SpikeCore/SpikeCore/MessageBus/IrcSendChannelMessage.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace SpikeCore.MessageBus
```
Comment (kog) [inline src/SpikeCore/SpikeCore/MessageBus/IrcSendChannelMessage.cs]: Can you please bubble this back through the modules?

**Comment by kog** [comment]
Looks like this also failed to build on Travis.

File: src/SpikeCore/SpikeCore.Irc.IrcDotNet/IrcClient.cs (hunk 1 above)
Comment (AndyCJ) [reply src/SpikeCore/SpikeCore.Irc.IrcDotNet/IrcClient.cs]: I have TODONE the TODO to TODONE TODO.

File: src/SpikeCore/SpikeCore/MessageBus/IrcSendChannelMessage.cs (hunk 2 above)
Comment (AndyCJ) [reply src/SpikeCore/SpikeCore/MessageBus/IrcSendChannelMessage.cs]: Done as part of fixing the build errors.

**Pull request merged**